likecoin/includes/class-likecoin-matters-api.php

```diff
@@ -89,10 +89,11 @@ public function login( $email, $password ) {
 		$request = wp_remote_post(
 			$this->base_url,
 			array(
-				'headers' => array(
+				'headers'    => array(
 					'Content-Type' => 'application/json',
 				),
-				'body'    => wp_json_encode(
+				'user-agent' => 'LikeCoin WordPress Plugin' . LC_PLUGIN_VERSION,
```

Comment:
Need to add a space before the version?
```suggestion
				'user-agent' => 'LikeCoin WordPress Plugin ' . LC_PLUGIN_VERSION,
```